integration_tests.py

```diff
@@ -33,3 +34,38 @@ def test_service(snap: Snap, service: str):
     assert service in services
     assert services[service].enabled
     assert services[service].active
+
+
+def test_config_set_unset(snap: Snap):
+    # Unset the test keys before running the test proper just in case
+    # it has been set by another test.
+    try:
+        snap.config.unset(['foo', 'goo'])
+    except UnknownConfigKey:
+        pass
```

Comment:
use `contextlib.suppress` for ease of reading?

```suggestion
    with suppress(UnknownConfigKey):
        snap.config.unset(['foo', 'goo'])
```